assembly/package-res/Kitchen.bat

```diff
@@ -1,7 +1,8 @@
 @echo off
+SET initialDir=%cd%
```

Comment:
Why are you doing this before the setlocal - I believe this needs to be done after the setlocal so that the variable doesn't stay in the user's environment after the script runs.